engine/src/main/scala/cromwell/server/CromwellRootActor.scala

```diff
@@ -122,6 +122,21 @@ abstract class CromwellRootActor(terminator: CromwellTerminator,
     .props(CopyWorkflowLogsActor.props(serviceRegistryActor, ioActor)),
     "WorkflowLogCopyRouter")
 
+  private val workflowCallbackConfig = WorkflowCallbackConfig(config.getConfig("workflow-state-callback"))
+
+  lazy val workflowCallbackActor: Option[ActorRef] = {
```

Comment:
It feels a little weird to make this actor arg optional, but it was the best way I thought of to minimize the impact when this feature isn't in use (ie in our large multi-tenant instance). I didn't want all Cromwell instances to have a `WorkflowCallbackActor` sitting around receiving no-op events whenever workflows finished.